cosmos/cheqd-mainnet.json

```diff
@@ -0,0 +1,52 @@
+{
+  "chainId": "cheqd-mainnet-1",
+  "chainName": "cheqd",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/cheqd/chain.png",
```

Comment:
Could you change this link to `https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/cheqd-mainnet/chain.png`

Reply:
cheqd -> cheqd-mainnet

Reply:
Ah yep, sorry for the inconvenience!